gratipay/utils/i18n.py

```diff
@@ -103,17 +104,19 @@ def get_function_from_rule(rule):
 
 
 def get_text(loc, s, *a, **kw):
+    escape = kw.pop('escape', escape_noop)
```

Comment:
That's prone to silent failure, we could potentially break escaping without noticing.